src/BDoc.cpp

```diff
@@ -490,6 +490,10 @@ void BDoc::parseManifestAndLoadFiles(const ZipSerialize &z, const vector<string>
     {
         THROW("Failed to parse manifest XML: %s (xsd path: %s)", e.what(), Conf::instance()->xsdPath().c_str());
     }
+    catch (...)
```

Comment:
If i remember correctly, throw in catch block without exception is rethrow? And the exception should be OutOfMemoryException